types/spotify-api/index.d.ts

```diff
@@ -874,7 +874,8 @@ declare namespace SpotifyApi {
         /**
          * The type of the album: one of “album”, “single”, or “compilation”.
          */
-        album_type: 'album' | 'single' | 'compilation';
+        album_type: 'album' | 'single' | 'compilation' |
+                    'ALBUM' | 'SINGLE' | 'COMPILATION';
```

Comment:
I think you found a "bug" in the spotify API. Requesting the same album returns the "album_type" in all lower case:
```
GET https://api.spotify.com/v1/albums/6wGcuJq8QjSeoYMJWgW7Wo
Authorization: Bearer {{access_token}}
Content-Type: application/json

###

{
  "album_type": "single",
  "artists": [
    {
      "external_urls": {
        "spotify": "https://open.spotify.com/artist/7jy3rLJdDQY21OgRLCZ9sD"
      },
      "href": "https://api.spotify.com/v1/artists/7jy3rLJdDQY21OgRLCZ9sD",
      "id": "7jy3rLJdDQY21OgRLCZ9sD",
      "name": "Foo Fighters",
[...]
```
But requesting the `/recommendations` yields the `album_type` in caps. I suggest adding a separate type for recommendations, as that seems to be the only endpoint that returns the type in this manner.